### DISCUSSION TITLE
Explanation different purposes of these 3 folders

### DISCUSSION THREAD

**Discussion by c1972** [thorsten/phpMyFAQ#3031] [Q&A] [answered]
Hello Thorsten :)

please be so kind to explain to me the **different purposes** of following 3 folders:
- /img/
- /images/
- /assets/themes/MYTHEME/img/

Thanks + regards Christian

**Comment by thorsten** [comment, marked as answer]
1. I don't know of the first one, it's not used and not from the phpMyFAQ project
2. It's for the uploaded files from the admin backend
3. It's for the images you need for your template theme, usually the Favicon and the logo

Short note: this will all change in 4.0

**Comment by c1972** [comment]
I think my question was really unnecessary (and stupid) this time.

I can't remember, but I probably created that third folder myself. 🙃

Sorry Thorsten!
